GANDLF/models/__init__.py

```diff
@@ -91,3 +93,35 @@
     "efficientnetb6": efficientnetB6,
     "efficientnetb7": efficientnetB7,
 }
+
+
+def get_modelbase_final_layer(final_convolution_layer):
```

Comment:
I think this function can be removed. Once this is done, this PR can be merged!

Reply:
Done, thanks and sorry for missing that. I am getting a formatting complaint from git on GANDLF/models/__init__.py, but I am running black on it and it is saying nothing needs to be changed. I'm not sure how to proceed with that.

Reply:
Thank you! I am not sure what complain git is giving but black is all good, so we can happily merge! 😄